src/FormField/DatePickerField.tsx

```diff
@@ -64,7 +55,7 @@ export class DatePickerField extends React.PureComponent<InternalProps> {
         hint={hint}
         render={(onFocus, onBlur) =>
           datePickerRenderer({
-            ...datePickerProps,
+            ...this.props,
```

Comment:
beware that here you're passing a lot of additional/unwanted props to the `DatePicker` underneath. Not sure this is something we want (and yes, spreading the DatePicker props at the top level wasn't really a good idea... we should change this for every field 🤔)

Reply:
True, but I had to do it because spreading and re-packing `this.props` was making it loose its conditional typing making it not usable as argument of `datePickerRenderer`. It's the easiest way and I think that now it creates no problem.
Otherwise if this is not acceptable I will find a way to extract `DatePicker.Props` from `DatePickerField.Props` preserving the typing

Reply:
oh god, you're right 🤦‍♂ 
Let's keep it like this atm. Could you please open a breaking issue to pack all these props inside a single `componentProps` in every field, instead of spreading them at top level?